MuteSpotifyAds/it.lproj/MainMenu.strings

```diff
@@ -9,22 +9,25 @@
 "DTq-sR-yDf.title" = "🔖 Licenza";
 
 /* Class = "NSMenuItem"; title = "≡  Song log"; ObjectID = "GPc-bM-QPt"; */
-"GPc-bM-QPt.title" = "≡  Log delle cazoni";
-
-/* Class = "NSMenuItem"; title = "© 2019 Simon Meusel"; ObjectID = "IUf-ND-5vk"; */
```

Comment:
Why did you remove this part of the file?

Reply:
git marks that as removed, it really is just in a different place, i believe because of the way the xcode localisation tool saved it

Reply:
now i'm noticing that it may have done this because the ObjectID for `Class = "NSMenuItem"; title = " ©  Simon Meusel";  ` was wrong, in fact before (IUf-ND-5vk) it didn't match the one in MainMenu.xib, now (vrP-PM-sCp) it does